### ISSUE TITLE
Choosing default visual style of geometric objects

### ISSUE THREAD

**Issue by joshwlambert** [epiverse-trace/tracetheme#3]
This issue is to discuss the default style of: line widths/types, axis labels and titles, point types/sizes and legend position.

After Visualisation task and finish group meeting two there was a general agreement that the defaults to all of the above aspects from {ggplot2} are sensible, and there is no need to make these defaults different without justification. 

Line widths/types and point types/sizes are defined within the `geom_*()` (unless there is a way to circumvent this), which means it is not possible to control these through the `theme_trace()`.

Some points raised in the meeting: 
1. Specifying a default legend position for all plots is difficult because for some plots having it above/below provides a clearer layout, but this is largely dependent on the data being plotted and the dimensions of the plot. We could decide on a default different from {ggplot2}, but that this should be changed when needed.
2. We prefer explicitly writing the y-axis title on the axis, rather than replacing it with a title or subtitle (as is common in data journalism).
3. Avoid using different point types/sizes for the sake of difference. One options for difference with the {ggplot2} default is open circles instead of filled, but avoid [other shapes](http://www.sthda.com/english/wiki/ggplot2-point-shapes).

Examples using these defaults are shown in https://epiverse-trace.github.io/tracetheme/articles/demo.html.

Fonts, which also fall under this topic, have been addressed in #2.

**Comment by Bisaloo** [comment]
Said differently and since there were no reactions to this issue, it seems that the only theming aspects we want to touch are discussed in #7.

Is this your understanding as well?

**Comment by joshwlambert** [comment]
Yes that's also my understanding. Please let me know if you would like this information recorded somewhere in the {tracetheme} package (e.g. `README` or vignette). Then we can close this issue.

**Comment by Bisaloo** [comment]
Yes, please, a "what the theme will not change" subsection in the design vignette, with the rationale behind these choices, would be helpful.
